tidb-cloud/serverless-limitations.md

```diff
@@ -8,49 +8,51 @@ aliases: ['/tidbcloud/serverless-tier-limitations']
 
 <!-- markdownlint-disable MD026 -->
 
-This document describes the limitations of TiDB Serverless.
+TiDB Serverless works with almost all workloads that TiDB supports, but there are feature differences between TiDB Self-Hosted or TiDB Dedicated clusters and TiDB Serverless clusters. This document describes the limitations of TiDB Serverless. 
 
 We are constantly filling in the feature gaps between TiDB Serverless and TiDB Dedicated. If you require these features or capabilities in the gap, use [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) or [contact us](https://www.pingcap.com/contact-us/?from=en) for a feature request.
 
 ## Limitations
 
-### SQL
+### Audit Logs
 
-- [Time to live (TTL)](/time-to-live.md) is not available for TiDB Serverless clusters currently.
-- The [`FLASHBACK CLUSTER TO TIMESTAMP`](/sql-statements/sql-statement-flashback-to-timestamp.md) syntax is not applicable to [TiDB Serverless](/tidb-cloud/select-cluster-tier.md#tidb-serverless-beta) clusters.
-- The [`SLEEP()` function](/functions-and-operators/miscellaneous-functions.md) only supports a maximum sleep time of 300 seconds.
+- Database audit logging is currently unavailable.
+- Console audit logging is currently unavailable.
```

Comment:
Please add a note on https://docs.pingcap.com/tidbcloud/tidb-cloud-console-auditing saying that it is unavailable for serverless

Reply:
Thanks for pointing out. Looks like this console auditing is available for serverless. It's fixed now.